src/Feature/Hover/Feature_Hover.re

```diff
@@ -23,6 +23,7 @@ type model = {
   triggeredFrom:
     option([ | `CommandPalette | `Mouse(EditorCoreTypes.Location.t)]),
   lastRequestID: option(int),
+  editor: option(Feature_Editor.Editor.t),
```

Comment:
I don't think you need to store the entire editor record here. Just the `id` should be sufficient. Storing the entire editor violates single source of truth as it can easily get stale/out-of-sync, which leads to bugs.

Reply:
This is also changed in my latest commit. Thanks again!